### PR TITLE
Disk hotplug

### LINKED ISSUE #35: VNC without password
Enable svncterm argument for password

https://github.com/OpenNebula/addon-lxdone/blob/7bc9b165e97e3a37488ba9169a2bb3ed186460ca/src/remotes/vmm/lxd/vnc.bash#L9

### PR DESCRIPTION
**Pull request by dann1**
Disk hot-plug comes online and also cleaner code.
fixes #47, fixes #35

### CODE REVIEW

**Comment by dann1** [comment]
- xml_query_item function got removed in the code merge. This was the main issue here.
- @jmdelafe I changed a bit your new updater, waiting for feedback.

**Comment by dann1** [comment]
- @jmdelafe I changed your new updater a bit, waiting for feedback
- xml_qurey_item method got deleted in the code merge, human error :)

File: src/remotes/vmm/lxd/lxd_common.py [hunk 1]
```diff
@@ -39,14 +39,13 @@
...
     sep = ': '
-    if severity == "e":
-        severity = "ERROR"
-    elif severity == "i":
-        severity = "INFO"
-    elif severity == "d":
-        severity = "DEBUG"
-
-    print(severity + sep + os.path.basename(sys.argv[0]) + sep + str(message), file=sys.stderr)
+    case = {
```
Comment (dann1) [inline src/remotes/vmm/lxd/lxd_common.py]: :+1:

File: src/remotes/vmm/lxd/lxd_common.py [hunk 2]
```diff
@@ -182,10 +195,12 @@ def storage_lazer(device):
 
 def storage_rootfs_mount(VM_ID, DISK_TYPE, DS_ID, DISK_SOURCE, DISK_CLONE):
     'Mounts rootfs for container one-$VM_ID'
-    source = storage_sysmap('0', DISK_TYPE, DISK_SOURCE, VM_ID, DS_ID, DISK_CLONE)
+    source = storage_sysmap('0', DISK_TYPE, DISK_SOURCE,
+                            VM_ID, DS_ID, DISK_CLONE)
```
Comment (jmdelafe) [inline src/remotes/vmm/lxd/lxd_common.py:199]: We assume the first disk (DISK_ID='0') is the rootfs of the container. What if the first disk isn't the rootfs?

**Review by jmdelafe — APPROVED** [review]
+1

File: src/remotes/vmm/lxd/lxd_common.py (hunk 2 above)
Comment (dann1) [reply src/remotes/vmm/lxd/lxd_common.py:199]: That could be improved, but so far it has always been :)

**Pull request merged**